lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenPluginUtils.java

```diff
@@ -43,17 +45,30 @@ private MavenPluginUtils() {
 		// Utility class, not meant to be instantiated
 	}
 
-	public static MarkupContent getMarkupDescription(Parameter parameter) {
+	public static MarkupContent getMarkupDescription(Parameter parameter, boolean supportsMarkdown) {
+		UnaryOperator<String> toBold = message -> supportsMarkdown ? MarkdownUtils.toBold(message) : message;
 		String description = parameter.getDescription();
 		description = htmlXMLToMarkdown(description);
-		return new MarkupContent("markdown",
-				"**required:** " + parameter.getRequirement() + LINE_BREAK + "**Type:** " + parameter.getType() + LINE_BREAK
-						+ "Expression: " + parameter.getExpression() + LINE_BREAK + "Default Value: " + parameter.getDefaultValue()
-						+ LINE_BREAK + description);
+		
+		String markdownDescription = 
+				toBold.apply("Required: ") + parameter.isRequired() + LINE_BREAK + 
```

Comment:
My previous mention about LINE_BREAK not depending on whether plaintext or markdown is used also applies here. @mickaelistria WDYT? Should lineBreak be a variable dependant on supportsMarkdown?

Reply:
yes.